soc/arm/nordic_nrf/Kconfig.defconfig

```diff
@@ -33,6 +33,11 @@ config SYS_POWER_MANAGEMENT
 config BUILD_OUTPUT_HEX
 	default y
 
+if !CORTEX_M_DWT
```

Comment:
Not sure if this is correct (i.e. we do not need the if !CORTEX_M_DWT:
The nRF SoCs _always have SOC_TIMING_FUNCTIONS and it should be default y.
The CMakeLists.txt , though, can make sure the ARCH (DWT) timing.c will be used, if present. Correct?